packages/salesforcedx-vscode-visualforce/src/extension.ts

```diff
@@ -54,14 +54,7 @@ export async function activate(context: ExtensionContext) {
 
   // The server is implemented in node
   const serverModule = context.asAbsolutePath(
-    path.join(
-      'node_modules',
-      '@salesforce',
-      'salesforcedx-visualforce-language-server',
-      'out',
-      'src',
-      'visualforceServer.js'
-    )
+    path.join(...context.extension.packageJSON.serverPath)
```

Comment:
Ok I see where that reference is coming from - why is this worth setting in the package.json?

Reply:
b/c it changes based on dev mode vs prod mode.  We def have a few other options we could consider like env vars or hardcoding in source.  Since I have to do some replaces in the package.json with packing anyway it seemed like as good a place as any.